lib/api/pokedex_api_spec/handler/pokemon_handler.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:pokedex/api/pokedex_api_spec/model/pokemon_model.dart';
+import 'package:dio/dio.dart';
+
+class PokemonHandler {
+  static Future<List<PokemonModel>?> getPokemonList() async {
+    var dio = Dio();
+    Response response = dio.get('') as Response;
+    try {
+      response = await dio.get('https://pokeapi.co/api/v2/pokemon/?limit=20');
```

Comment:
Can you create constants for this like
baseApi =  https://pokeapi.co
and others for parameters